osutil/flock_test.go

```diff
@@ -211,6 +211,7 @@ func (s *flockSuite) TestLockUnlockNonblockingWorks(c *C) {
 	sleeperKillerPath := filepath.Join(c.MkDir(), "pid")
 	// we can't use --no-fork because we still support 14.04
 	cmd := exec.Command("flock", "--exclusive", lockPath, "-c", fmt.Sprintf(`echo "kill $$" > %s && exec sleep 30`, sleeperKillerPath))
+	cmd.Env = append(cmd.Env, "SHELL=")
```

Comment:
Great catch! Perhaps a comment in the code here would be helpful to know why we're unsetting the variable.